themes/bootstrap3/templates/search/searchTabs.phtml

```diff
@@ -2,7 +2,7 @@
   <ul class="nav nav-tabs">
     <?php foreach ($searchTabs as $tab): ?>
         <?php if ($this->permission()->allowDisplay($tab['permission'])): ?>
-          <li<?=$tab['selected'] ? ' class="active"' : ''?>>
+          <li<?=$tab['selected'] ? ' class="active"' : ($showCounts ? ' class="show-counts"' : '')?>>
```

Comment:
Would it be better to use a `data-show-counts` attribute instead of a class, since this is not really a style-related issue?

Reply:
Yes.